docs: add from which version public_metrics start (#671)
redpanda-data/documentation: modules/reference/pages/rpk/rpk-registry/rpk-registry-mode-get.adoc

= rpk registry mode get
// tag::single-source[]

> Get Schema registry's mode.

Reviewer: ```suggestion
Check the mode Schema Registry is in.
```

> Running this command with no subject returns the global mode, alternatively you can use the `--global` flag to get the global mode at the same time as per-subject modes.

Reviewer: ```suggestion
Running this command with no subject returns the global mode for all schemas managed by Schema Registry. Alternatively, use the `--global` flag to return the global mode at the same time as per-subject modes.
```
@r-vasquez - Is the description of global mode in the first line of this suggestion correct?

Reviewer 2: You can set a mode either globally, or on a per-subject basis. The Global mode is like a default, so, if a schema does not have a per-subject mode, it will default to the global mode.

Saying: ` for all schemas managed by Schema Registry.` may imply incorrectly that the global mode overrides the per-subject mode, what do you think?

Author: All schemas are managed by SR, no?

Author: ```suggestion
Running this command with no subject returns the global mode for Schema Registry. Alternatively, use the `--global` flag to return the global mode at the same time as per-subject modes.
```

Reviewer: @Deflaimun, I've updated both descriptions closer to the original as agree with @r-vasquez that my original change might be misleading.

> Running this command with no subject returns Schema Registry's global mode. Alternatively, use the `--global` flag to return the global mode at the same time as per-subject modes.

Reviewer: ```suggestion
Running this command with no subject returns the global mode for Schema Registry. Alternatively, use the `--global` flag to return the global mode at the same time as per-subject modes.
```
